### ISSUE TITLE
PresentrTransitionContext properties are internal and thus cannot be used when subclassing PresentrAnimation

### ISSUE THREAD

**Issue by MilanNosal** [IcaliaLabs/Presentr#96]
I am trying to create a custom transition animation. However, PresentrTransitionContext properties have internal accessibility level and thus are not accessible from my subclass.
